public/components/map_page/map_page.tsx

```diff
@@ -58,25 +81,29 @@ export const MapPage = ({ mapConfig }: Props) => {
         setLayersIndexPatterns(savedIndexPatterns);
       });
     } else {
-      const initialDefaultLayer: MapLayerSpecification = getLayerConfigMap(mapConfig)[
-        OPENSEARCH_MAP_LAYER.type
-      ];
+      const initialDefaultLayer: MapLayerSpecification =
+        getLayerConfigMap(mapConfig)[OPENSEARCH_MAP_LAYER.type];
       initialDefaultLayer.name = MAP_LAYER_DEFAULT_NAME;
       setLayers([initialDefaultLayer]);
     }
   }, []);
```

Comment:
nit: if [] is empty.. could we just write

```
useEffect(() => {...});
```

```

Reply:
updated